apartments/tests.py

```diff
@@ -118,32 +80,29 @@ def test_apartment_form_validity(city, address, rent, num_of_roomates, num_of_ro
     assert form.is_valid() is validity
 
 
-@pytest.fixture
-def valid_form(new_city):
-    return ApartmentCreationForm(data={
-        'city': new_city,
-        'address': 'address',
-        'rent': 10,
-        'num_of_roomates': 2,
-        'num_of_rooms': 2,
-        'start_date': 2020-1-1,
-        'about': 'about',
-    })
-
-
 @pytest.mark.django_db
-def test_valid_form_is_valid(valid_form):
-    assert valid_form.is_valid
+def test_valid_form_is_valid(valid_apartment_creation_form):
+    assert valid_apartment_creation_form.is_valid
 
 
 @pytest.mark.django_db
-def test_fail_to_save_apartment_form_with_commit_true(valid_form):
+def test_fail_to_save_apartment_form_with_commit_true(valid_apartment_creation_form):
     with pytest.raises(ValueError):
-        valid_form.save(commit=True)
+        valid_apartment_creation_form.save(commit=True)
```

Comment:
Is the `valid_apartment_creation_form` not valid? Why is it supposed to raise an exception?

Reply:
With commit=True it's supposed to save the Apartment immediately to the data-base, because the form does not have a `owner` field I made it so the form throws an exception when you try to save it with commit=True, this test tests it